hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/S3SecretValue.java

```diff
@@ -38,16 +38,29 @@ public static Codec<S3SecretValue> getCodec() {
   }
 
   // TODO: This field should be renamed to accessId for generalization.
-  private String kerberosID;
-  private String awsSecret;
-  private boolean isDeleted;
-  private long transactionLogIndex;
+  private final String kerberosID;
+  private final String awsSecret;
```

Comment:
An unrelated comment: It is not a good idea store a secret in a plaintext `String` and write it to db.

The usual practice is to encrypt it in `byte[]` and erase it after use.